bazel/repository_locations.bzl

```diff
@@ -1130,13 +1130,13 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "VPP Comms Library",
         project_desc = "FD.io Vector Packet Processor (VPP) Comms Library",
         project_url = "https://fd.io/",
-        version = "596c45b22211c9af243b624dc037f58c0aa1c302",
-        sha256 = "e4c3fad7e1a6952e5c081cfe25f1f091d97fae8e75c5f03205def37d34c27741",
+        version = "7c3275e84b64ade4e20d00e4457bd4e437b1894f",
+        sha256 = "d456d37bbb7f90ec1ef166c1387e788b4c3078d38303f12ab41f1d0ac1a1cfc0",
         strip_prefix = "vpp-{version}",
         urls = ["https://github.com/FDio/vpp/archive/{version}.tar.gz"],
         use_category = ["other"],
         extensions = ["envoy.bootstrap.vcl"],
-        release_date = "2021-09-13",
+        release_date = "2021-12-10",
```

Comment:
Can we switch to using release tags for VPP? It looks like this commit isn't in a released tag yet but using tags would be preferable - https://github.com/FDio/vpp/tags

Reply:
Fair point but the vcl socket interface still under development so for the time being I would prefer to avoid latching on to VPP releases which are done every 4 months.